src/utils.js

```diff
@@ -208,3 +211,21 @@ export const convertToSlipData = (source, intl, timeZone, locale, slipName = 'Pi
     };
   });
 };
+
+export function buildUrl(location, values) {
+  let url = values._path || location.pathname;
+  const locationQuery = location.query ? location.query : queryString.parse(location.search);
+  const params = {};
+
+  forOwn(Object.assign(locationQuery, values), (value, key) => {
```

Comment:
Could we use `omit` here instead? https://lodash.com/docs/4.17.15#omit

Reply:
Done